litmus-portal/frontend/src/components/ToggleComponent/index.tsx

```diff
@@ -1,69 +1,80 @@
-import { Typography } from '@material-ui/core';
-import ToggleButton from '@material-ui/lab/ToggleButton';
-import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
 import React from 'react';
+import { Typography } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import useStyles from './styles';
 
 const ToggleComponent = () => {
-  const [alignment, setAlignment] = React.useState<string | null>('pass');
+  const classes = useStyles();
+
+  // Default Props are false
+  const [currentPassState, setCurrentPassState] = React.useState<
```

Comment:
This code can be refactored to use only 1 boolean state var. There are only 2 possible values pass or fail(binary), so 1 var should be enough to handle it.

Reply:
Yup, I'll take a look into this

Reply:
@gdsoumya  Sayan has fixed this issue and will be sending a PR with some changes.